src/client/buttons/HomeButton/HomeButton.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import IconButton from '../IconButton';
+
+const HomeButton = (props) => {
+  const { className, onClick } = props;
+  return (
+    <IconButton
+      icon=""
```

Comment:
is there a reason why `icon=""`? 
is this required from the doc or is it displaying wrongly for me?

Reply:
Yeap just checked it. It’s an emoji, but it was displaying wrong on my browser. So dw about it